src/matching/Generic_vs_generic.ml

```diff
@@ -1218,6 +1257,15 @@ and m_wrap_m_int_opt (a1, a2) (b1, b2) =
       let b1 = Parse_info.str_of_info b2 in
       m_wrap m_string (a1, a2) (b1, b2)
 
+and m_wrap_m_string_opt (a1, a2) (b1, b2) =
+  match (a1, b1) with
+  (* iso: semantic equivalence of value! 0x8 can match 8 *)
```

Comment:
You've copied the comment from below but this actually does not apply here ... "0x8" will not match "8" ...